package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A super simple JS library for playing sound effects and other audio. Now with button mashing support!",
   "main": "soundbox.mjs",
   "scripts": {
-    "test": "echo How do you automatically test a browser-based sound player? I have no idea...."
+    "test": "echo How do you automatically test a browser-based sound player? I have no idea....",
+    "serve": "http-server example -o"
```

Comment:
If I try to execute the `http-server` command in my terminal manually, I get a command not found:

```bash
$ http-server --help
http-server: command not found
```

Could you perhaps explain where this comes from please? It's not particularly intuitive.

I see you've got this `http-server` package as a dev dependency, but since that's a local package if this is executed in a user's normal shell you should still get a command not found. Shouldn't this call `node_modules/.bin/http-server` instead?

Reply:
If you installed the npm package http-server globally then it'll work.

```
npm install -g http-server
http-server --help
``` 
But you don't need to install it globally if you use the new `npx` binary included in standard Nodejs 8.x+ installations.

```
npm install http-server
npx http-server --help
```

Lastly, here's the reason why the http-server command works in the npm scripts section without being globally installed.

> In addition to the shell’s pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix. For example, if there is a devDependency on tap in your package, you should write:
> 
> "scripts": {"test": "tap test/\*.js"}
> instead of
> 
> "scripts": {"test": "node_modules/.bin/tap test/\*.js"}  
> to run your tests.
> &mdash; <cite>[Run script docs][1]</cite>

[1]:https://docs.npmjs.com/cli/run-script
- npx docs: https://www.npmjs.com/package/npx

Reply:
Ah, I see! Thanks for the explanation. I've seen `npx`, but I haven't actually used it yet.